lib/tapping_device/payload.rb

```diff
@@ -0,0 +1,23 @@
+class TappingDevice
+  class Payload < Hash
+    ATTRS = [:receiver, :method_name, :arguments, :return_value, :filepath, :line_number, :defined_class, :trace, :tp]
+
+    ATTRS.each do |attr|
+      define_method attr do
+        self[attr]
+      end
+    end
+
+    def self.init(hash)
```

Comment:
`Hash.new({foo: bar})` won't work as expected. We need to initialize an empty hash and assign k/v pairs manually.